src/Bridge/Anthropic/ModelHandler.php

```diff
@@ -55,13 +61,28 @@ public function request(Model $model, object|array|string $input, array $options
         ]);
     }
 
+    /**
+     * @throws TransportExceptionInterface
+     * @throws ServerExceptionInterface
+     * @throws RedirectionExceptionInterface
+     * @throws DecodingExceptionInterface
+     * @throws ClientExceptionInterface
+     */
     public function convert(ResponseInterface $response, array $options = []): LlmResponse
     {
         if ($options['stream'] ?? false) {
             return new StreamResponse($this->convertStream($response));
         }
 
-        $data = $response->toArray();
+        $data = $response->toArray(false);
```

Comment:
let's revert this change but keep the structural checks below

```suggestion
        $data = $response->toArray();
```